src/index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8" />
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <title>Leaderboard</title>
+  <link rel="icon" type="image/png" href="favicon.ico" />
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet"
+    integrity="sha384-EVSTQN3/azprG1Anm3QDgpJLIm9Nao0Yz1ztcQTwFspd3yD65VohhpuuCOmLASjC" crossorigin="anonymous" />
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-MrcW6ZMFYlzcLA8Nl+NtUVF0sA7MsXsP1UyJoMp4YLEuNSfAP+JcXn/tWtIaxVXM"
+    crossorigin="anonymous"></script>
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.5.0/font/bootstrap-icons.css">
+</head>
+
+<body class="d-flex flex-column min-vh-100">
+  <header>
+    <h1 class="m-5">Leaderboard</h1>
+  </header>
+  <div class="container">
+    <div class="row">
+      <div class="col">
+        <div class="d-flex justify-content-between py-5">
+          <h2>Recent Scores</h2>
+          <button type="button" class="btn btn-light">Refresh</button>
```

Comment:
Good job

- [ ] Kindly remove the duplicate, you have the same content in both your main and the one in the dist folder, your main index.html should contain only one root element.
<img width="288" alt="Screenshot 2021-07-27 at 23 09 07" src="https://user-images.githubusercontent.com/34798898/127239067-b3fb2447-08e1-40a2-9b86-170d248404df.png">

Reply:
I'm sorry but that's not a requirement.
![image](https://user-images.githubusercontent.com/14119855/127239491-c1acace3-7e19-4f6c-a17a-d9fcbd827584.png)

Reply:
thanks, I guess you meant the dist/index.html file to be in production mode.